Sources/FluentPostgresDriver/FluentPostgresConfiguration.swift

```diff
@@ -2,6 +2,7 @@ extension DatabaseConfigurationFactory {
     public static func postgres(
         url urlString: String,
         maxConnectionsPerEventLoop: Int = 1,
+        newConnectionTimeout: NIO.TimeAmount = .seconds(10),
```

Comment:
Saying "new connection" is not technically correct here. The timeout is for the `requestConnection()` method on connection pools which may return a new or existing connection.

Perhaps `connectionRequestTimeout` or `connectionPoolTimeout` would be better?

Reply:
Good point. I initially steered away from the word "request" because I wanted it to be clear that the timeout being set was not on the requests made against the connection but rather on the pool, but both of your suggestions do an adequate job of disambiguating.

I think I am leaning toward `connectionPoolTimeout` because it still eschews the word "request." Will update.